internal/cli/command/controlplane/init.go

```diff
@@ -339,8 +343,10 @@ func runInit(options initOptions, banzaiCli cli.Cli) error {
 		if out[externalHost] == nil {
 		}
 
-		if options.installerImageRepo == defaultImage && options.provider == providerCustom {
-			return errors.New("Custom provisioning is available by specifying a custom installer image. Please refer to your deployment guide or use one of our support channels.")
+		if options.provider == providerCustom {
+			if hasExports && !imageMeta.Custom.Enabled || !hasExports && options.installerImageRepo == defaultImage {
```

Comment:
Why is the ability to disable an image to be used with the CLI good for us? I'm sure I'm forgetting something obvious here...